packages/jest-runner/corpus.ts

```diff
@@ -56,6 +66,38 @@ const buildInputsDirectory = (
 	return path.join(root.dir, root.name, ...pathElements) + path.sep;
 };
 
+/**
+ * Returns the path to the corpus directory for the given test file and test descriptions.
+ */
+const buildCorpusDirectory = (
```

Comment:
I wonder if common code from `buildInputsDirectory` and `buildCorpusDirectory`could be extracted into a dedicated function.